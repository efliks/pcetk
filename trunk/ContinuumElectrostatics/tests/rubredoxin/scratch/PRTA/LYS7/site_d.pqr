ATOM      0  CE  LYS     7      11.855  42.933  -5.846   0.10000   1.700      0      0      0      0
ATOM      0  HE1 LYS     7      11.006  43.606  -5.618   0.00000   1.000      0      0      0      0
ATOM      0  HE2 LYS     7      11.850  42.093  -5.122   0.00000   1.000      0      0      0      0
ATOM      0  NZ  LYS     7      11.633  42.350  -7.229  -0.25000   1.550      0      0      0      0
ATOM      0  HZ1 LYS     7      11.153  41.430  -7.105   0.05000   0.300      0      0      0      0
ATOM      0  HZ2 LYS     7      12.559  42.178  -7.703   0.05000   0.300      0      0      0      0
ATOM      0  HZ3 LYS     7      11.031  42.990  -7.777   0.05000   0.300      0      0      0      0
