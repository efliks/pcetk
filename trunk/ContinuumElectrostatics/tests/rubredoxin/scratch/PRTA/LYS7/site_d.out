Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/LYS7//site_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (13.319, 37.4012, 4.51304)) 121 2
Finer level:    (11.5839, 42.5114, -6.62863) 101 1
Finer level:    (11.5839, 42.5114, -6.62863) 101 0.5
Finest level:   (11.5839, 42.5114, -6.62863) 101 0.25


Self energy of scratch/PRTA/LYS7//site_d in three dielectric environment = 15.2516
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/LYS7//site_d with background charge set in three dielectric environment = -0.243967
(probably in kcal/mole)



Interaction energies of scratch/PRTA/LYS7//site_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.000563212
0   1 : 1.31038e-05
1   0 : -0.000728838
1   1 : -0.039228
2   0 : 53.8695
2   1 : 30.5039
3   0 : -0.00115074
3   1 : -0.00791718
4   0 : -0.000471428
4   1 : -0.00333153
5   0 : -0.000159847
5   1 : -0.00270127
6   0 : 6.96831e-05
6   1 : -0.000830405
7   0 : 0.000272425
7   1 : -0.000924557
8   0 : 2.26033e-05
8   1 : -0.000220098
9   0 : 0.000114506
9   1 : -0.00103808
10   0 : 1.57882e-05
10   1 : -0.00019541
11   0 : -5.84612e-05
11   1 : 3.32057e-06
12   0 : -2.28848e-05
12   1 : 6.50675e-05
13   0 : -1.31485e-05
13   1 : -3.73565e-05
14   0 : 1.46971e-05
14   1 : 0.000156476
15   0 : 2.35807e-06
15   1 : -0.000158667
16   0 : 0.000728647
16   1 : -0.013574
17   0 : -0.00120239
17   1 : -0.00784121
18   0 : 0.000885318
18   1 : 8.56505e-05
19   0 : 0.00206466
19   1 : -0.0110827
20   0 : -0.00242932
20   1 : -0.0146079
21   0 : -0.000106564
21   1 : 3.4728e-05
22   0 : -3.64041e-06
22   1 : 0.000461306
23   0 : 4.86566e-05
23   1 : 0.000329907
Total runtime: 15.11 s user 0.13 s system 
